nilearn/image/tests/test_image.py

```diff
@@ -271,3 +272,13 @@ def test_swap_img_hemispheres():
         data)
 
 
+def test_concat_imgs():
+    # simply smoke-test the import from _utils.concat_niimgs
```

Comment:
You could do something like this that would test that they are actually the exact same function

```python
# in the imports
from ..._utils import niimg_conversions

# in the tests
def test_concat_imgs():
    image.concat_imgs is niimg_conversions.concat_niimgs
```